common/djangoapps/django_comment_common/signals.py

```diff
@@ -0,0 +1,12 @@
+"""Signals related to the comments service."""
+
+from django.dispatch import Signal
+
+
+thread_created = Signal(providing_args=['user'])
+thread_edited = Signal(providing_args=['user'])
+thread_voted = Signal(providing_args=['user'])
+comment_created = Signal(providing_args=['user'])
```

Comment:
I wonder if we really want to differentiate between comments and threads. I suppose it gives more flexibility for the future, but it means registering for a lot of different things.